lib/capistrano/tasks/sidekiq.rake

```diff
@@ -276,4 +277,10 @@ namespace :sidekiq do
       properties.fetch(:run_as) || # global property across multiple capistrano gems
       role.user
   end
+
+  def sidekiq_config(role)
+    sidekiq_roles = Array.wrap(fetch(:sidekiq_roles)) || []
```

Comment:
`.wrap` is rails specific API, won't be working for non Rails App.

On top of that,   `|| []` is redundant. 

`Array(fetch(:sidekiq_roles))`   would be enough.